### DISCUSSION TITLE
stats are not display (windows-amd64) (linux-amd64)

### DISCUSSION THREAD

**Discussion by RFHACKE** [projectdiscovery/naabu#950] [General]
### Naabu version:
2.1.6 windows amd64
### Current Behavior:
when using naabu the stats are not displayed on http://localhost:63636
i also tried setting the stats option to true but did not work
### Expected Behavior:
should display the stats on  http://localhost:63636

### Steps To Reproduce:
Example: steps to reproduce the behavior:
1. Run 'naabu ..'
2. open http://localhost:63636
3. see 404 not found error

### Anything else:

![problem](https://github.com/projectdiscovery/naabu/assets/43932773/f186cf0d-d486-4c8a-9492-cbdc4819490b)

**Comment by RFHACKE** [comment]
ok i tryed this on linux amd64 but the respones is the same.

**Comment by SalahMSaleh** [comment]
any fix?

**Comment by pdelteil** [comment]
What command params are you using?

**Comment by RFHACKE** [comment]
@pdelteil sorry for late reply the command used was:
naabu -host {ipaddress} -tp full -o file.txt
i even tried changing the matrics port to 645 
naabu -host {ip} -tp full -o file.txt -mp 645
also tried with the stats flag
naabu -host {ip} -tp full -o file.txt -mp 645 -stats
still the response was same

**Comment by Mzack9999** [comment]
Please see https://github.com/projectdiscovery/naabu/discussions/947#discussioncomment-8305221

**Comment by RFHACKE** [comment]
@Mzack9999 ok thanks this resolves my issue but shouldn't they include this information in the readme.The readme only tells about the 
url (http://127.0.0.1:63636) and not directory (metrics) where result are displayed.The whole url with directory should be in the readme.

**Comment by RFHACKE** [comment]
@Mzack9999 and why was the old scan progress logging method removed.It was good enough as it is.
